internal/provider/provider.go

```diff
@@ -53,6 +53,11 @@ func (p *btpcliProvider) Schema(_ context.Context, _ provider.SchemaRequest, res
 				Optional:            true,
 				Sensitive:           true,
 			},
+			"idtoken": schema.StringAttribute{
+				MarkdownDescription: "A valid id token. To be provided instead of 'username' and 'password'. This can also be sourced from the `BTP_IDTOKEN` environment variable.",
```

Comment:
Please add (SAP-internal usage only) to this description. Although the fetching of the token can be done by customers, this is not the preferred way for authentication and is not yet available on all landscapes. As we need to create a new release before roll-out to all landscapes, this will safeguard customers from accidentally using the  parameter.